.happy/terraform/envs/rdev/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output frontend_url {
+  value       = module.stack.frontend_url
+  description = "The URL endpoint for the website service"
```

Comment:
do you think it's too redundant to change the description to this: "The URL endpoint for the frontend website service"